x-pack/legacy/plugins/spaces/public/lib/spaces_manager.ts

```diff
@@ -105,7 +108,15 @@ export class SpacesManager {
   }
 
   private async refreshActiveSpace() {
+    // Anonymous paths (such as login/logout) should not request the active space under any circumstances.
+    if (this.isAnonymousPath()) {
```

Comment:
**Note** This is a rather targeted fix; Once we have the remainder of the spaces client plugin migrated to the NP, we can make this more explicit, rather than an implementation detail of this manager.